server/fishtest/views.py

```diff
@@ -246,9 +246,23 @@ def nns(request):
 def actions(request):
     search_action = request.params.get("action", "")
     search_user = request.params.get("user", "")
+    search_before = request.params.get("before", None)
+    if search_before is not None:
```

Comment:
The `try ... except` works fine for `search_before is None`, so if I'm not wrong we could avoid the outer `if .. else`

Reply:
I removed the outer try except. Not tested since my local fishtest is a bit broken now.

Reply:
An example of an URL

https://dfts-0.pigazzini.it/actions?count=1&before=1635700115.683

Reply:
```
>>> float(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: float() argument must be a string or a real number, not 'NoneType'
```